### PR TITLE
Fix .de whois parsing

### LINKED ISSUE #104: .de parse error
Since GDPR, the content of .de TLDs is very limited and the format has changed. 
But at least the name servers could be parsed.

### PR DESCRIPTION
**Pull request by and0x000**
fixes #104

### CODE REVIEW

**Comment by and0x000** [comment]
@weppos please see recent commit for updated tests and fixtures

File: spec/whois/parsers/responses/whois.denic.de/de/status_registered_spec.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@
...
+      expect(subject.disclaimer).to eq("The above data may only be used within the scope of technical or administrative necessities of Internet operation or to remedy legal problems. The use for other purposes, in particular for advertising, is not permitted. The DENIC whois service on port 43 doesn't disclose any information concerning the domain holder, general request and abuse contact. This information can be obtained through use of our web-based whois service available at the DENIC website: http://www.denic.de/en/domains/whois-service/web-whois.html")
```
Comment (weppos) [inline spec/whois/parsers/responses/whois.denic.de/de/status_registered_spec.rb]: Please do not make changes to the spec directly. These files are auto-generated as explained at the top of the file.
You should edit the corresponding .expected file, and regenerate the specs with

```
rake spec:generate
```

Comment (and0x000) [reply spec/whois/parsers/responses/whois.denic.de/de/status_registered_spec.rb]: fixed with https://github.com/weppos/whois-parser/pull/111/commits/91685120e4a0b1fc66eba4376a366b8fd48f3e11

**Comment by and0x000** [comment]
closed due to lack of response from project owner.

**Pull request closed without merging**